client/src/slate-helpers/slate-utils/isSelectionAcrossPointers.js

```diff
@@ -0,0 +1,16 @@
+import { closestPointerExportAncestor } from "./closestPointerExportAncestor";
+
+export function isSelectionAcrossPointers(selection, document) {
+  const { anchorKey, focusKey } = selection;
+  const anchorNode = document.getNode(anchorKey);
+  const focusNode = document.getNode(focusKey);
+
+  const closestPointerExportAncestorOfAnchorNode = closestPointerExportAncestor(anchorNode, document);
+  const closestPointerExportAncestorOfFocusNode = closestPointerExportAncestor(focusNode, document);
+
+  if (closestPointerExportAncestorOfAnchorNode !== closestPointerExportAncestorOfFocusNode) {
+    return true;
+  }
+
+  return false;
```

Comment:
I find the above if / return / return pattern a bit awkward. Would prefer something like,
```js
return (closestPointerExportAncestorOfAnchorNode !== closestPointerExportAncestorOfFocusNode)
```